css/tablesort.css

```
@@ -0,0 +1,31 @@
+th[role=columnheader]:not(.no-sort) {
+    cursor: pointer;
+}
+
+th[role=columnheader]:not(.no-sort):after {
+    content: '';
+    float: right;
+    margin-top: 10px;
+    border-width: 0 4px 4px;
+    border-style: solid;
+    border-color: #404040 transparent;
```

```suggestion
    border-color: currentColor transparent;
```
This might work better dark-mode styled tables?

---

Since it's only shown on hover, this may be better

---

Not needed anymore (using entities now and they inherit the color)